bin/run-test.sh

```diff
@@ -15,6 +15,8 @@ function run_test() {
     meta=".meta"
     exercise_name="${1}"
     exercise_path="${exercises_path}/${exercise_name}"
+
+    echo "$exercise_name / $exercise_path"
```

Comment:
Since we `echo` the exercise name below, can we remove this?

Reply:
Ah, thanks for catching this – it was a debug echo that slipped into the PR